blueocean-bitbucket-pipeline/src/main/java/io/jenkins/blueocean/blueocean_bitbucket_pipeline/cloud/BitbucketCloudScm.java

```diff
@@ -13,9 +13,9 @@
  * @author Vivek Pandey
  */
 public class BitbucketCloudScm extends AbstractBitbucketScm {
+    public static final String ID = "bitbucket-cloud";
+    public static final String API_URL = "https://bitbucket.org";
     static final String DOMAIN_NAME="blueocean-bitbucket-cloud-domain";
-    static final String ID = "bitbucket-cloud";
-    static final String API_URL = "https://bitbucket.org/api/2.0/";
```

Comment:
This didn't appear to be used anywhere and the `serverUrl` returned from the Bitbucket Branch Source api was actually just the protocol+domain, so I changed it.